source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -216,6 +218,29 @@ function breadcrumb_trail_items_for_hooks( $items, $args ) {
 	return $items;
 }
 
+/**
+ * Remove the 'Reference' part of the breadcrumb trail.
+ *
+ * @param  array $items The breadcrumb trail items.
+ * @param  array $args  Original args.
+ * @return array
+ */
+function breadcrumb_trail_items_remove_reference( $items, $args ) {
+	if ( ! is_singular() && ! is_single() && ! is_post_type_archive() && ! is_archive() ) {
+		return $items;
+	}
+
+	return array_filter(
+		$items,
+		function( $item ) {
+			// Remove the 'reference' parent based on the presence of its URL.
+			// We can't use the label because of internationalization.
+			$result = (bool) preg_match( '/href=".*\/reference\/"/', $item, $matches );
```

Comment:
@dd32 I'm not sure if you are familiar with this code, but any opinion on pulling out the "reference" link from the breadcrumbs like this? 

I looked through the code that adds it, but I'm fearful of making changes during this "site refresh".

Reply:
For readability I'd use this regex:
```suggestion
			$result = (bool) preg_match( '!href="[^"]+/reference/"!', $item );
```

I don't see any issues with removing the item this way though.